Create ui-c8y-1018-503-149-data-broker-connector-does-not-have-a-warning-verification-message-when.md (#3092)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1018-503-149-data-broker-connector-does-not-have-a-warning-verification-message-when.md

---
date: ""

> title:Added confirmation dialog when deleting a data broker connector

Reviewer: ```suggestion
title: Added confirmation dialog when deleting a data broker connector
```

> title: A warning message is shown when deleting a data broker connector

Reviewer: ```suggestion
title:Added confirmation dialog when deleting a data broker connector
```

> title: DataBroker connector does not have a warning verification message when deleting it (#8410) [GRAFT][release/y2024] (#8476)

Reviewer 2: ```suggestion
title: A warning message is shown when deleting a data broker connector
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62022
version: 1018.503.149
---

> DataBroker connector does not have a warning verification message when deleting it (#8410) [GRAFT][release/y2024] (#8476)

Reviewer 2: ```suggestion
When attempting to delete a data broker connector from a connectors list, a warning message will be displayed asking the user to confirm the deletion. This helps prevent accidental deletion of data broker connectors.
```

> When attempting to delete a data broker connector from a connectors list, a warning message will be displayed asking the user to confirm the deletion. This helps prevent accidental deletion of data broker connectors.

Reviewer: ```suggestion
In the data broker connector configuration, there was no warning message displayed when attempting to delete a connector, which could lead to accidental deletions. This change introduces a confirmation dialog that is shown when trying to delete a data broker connector, ensuring that connectors are no longer unintentionally removed.
```
